components/emailoctopus/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/emailoctopus",
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
I think this should be `0.1.0` [since it introduces a new component](https://pipedream.com/docs/components/guidelines/#versioning)